### PR TITLE
Multiple categories

### PR DESCRIPTION
**Pull request by ratkinson0405**

### CODE REVIEW

**Review by marcodelapierre — CHANGES_REQUESTED** [review]
Hi Rob, thanks for your work!
I have only some comments on the resulting required syntax in the YAML, see below.

I find this syntax a bit too wordy in two regards:
```
topic: 
     - name: Supercomputing
     - name: Data
 ```
1. can we ditch the key: value syntax and have:
```
topic: 
     - Supercomputing
     - Data
 ```
Hasn't HTML got a "list" type of object, which can be iterated?
For instance, is it allowed to write default.html, line 98 as: `{{ topic }}<br>`  ?

If this were possible, the yaml writing would become more concise.

2. can we not use the list for just one item? (this is probably already possible, haven't tested)

Thanks,

**Comment by ratkinson0405** [comment]
Marco, I've made the changes to the topic items and removed the name tags. Can you please review? Thanks

**Review by marcodelapierre — APPROVED** [review]
good to go!

out of curiosity, is `- topic: ` required, or would `topic: ` also work?

**Comment by ratkinson0405** [comment]
OK, the hyphens are now removed.
